src/ChunkedStreamDecoder.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace React\HttpClient;
+
+use Evenement\EventEmitterTrait;
+use React\Stream\DuplexStreamInterface;
+use React\Stream\Util;
+
+class ChunkedStreamDecoder
```

Comment:
This should probably implement `ReadableStreamInterface`? (e.g. https://github.com/clue/php-utf8-react/blob/master/src/Sequencer.php#L13)